php/src/Core/Database/AbstractRepository.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Core\Database;
+
+use App\Core\Entities\AbstractDTO;
+use Cassandra\Rows;
+
+abstract class AbstractRepository
+{
+
+    /** @var string */
+    public $table = '';
+
+    /** @var string */
+    public $primaryKey = '';
+
+    /** @var Connector */
+    public $connection;
+
+    /** @var array */
+    public $keys = [];
+
+    public function __construct(Connector $connector)
+    {
+        $this->connection = $connector;
+    }
+
+    public function getById(string $id): Rows
+    {
+        $query = sprintf("SELECT * FROM %s WHERE %s = %s", $this->table, $this->primaryKey, $id);
+
+        return $this->connection
+            ->prepare($query)
+            ->execute()
+            ->get(5);

```

Comment:
@DanielHe4rt Why we have 5 here? Can we put this in a constant to [replace Magic Numbers](https://refactoring.guru/pt-br/replace-magic-number-with-symbolic-constant)?

Reply:
I knew it that you will not let it pass xDxD

I'm going to fix it on next commit.